lib/components/artist/shows/metadata.js

```diff
@@ -10,13 +10,12 @@ import SerifText from '../../text/serif'
 class Metadata extends React.Component {
 
   render() {
-    // debugger;
     return (
       <View style={styles.container}>
         <Text style={styles.sansSerifText}>{this.props.show.partner.name.toUpperCase()}</Text>
         <Text style={styles.sansSerifText}>{this.showTypeString(this.props.show)}</Text>
         <SerifText style={styles.serifText}>{this.props.show.name}</SerifText>
-        <SerifText style={styles.serifText, {color: 'grey'}}>{this.dateAndLocationString(this.props.show)}</SerifText>
+        { this.dateAndLocationString(this.props.show) }
```

Comment:
So if-else statements don't work in JSX: https://facebook.github.io/react/tips/if-else-in-JSX.html, but this method is. Basically, the centering was off because the parent was accounting for an invisible line of text when `dateAndLocationString` was null. This way, the `SerifText` never exists unless it's supposed to, so centering works as expected.